src/reports/models/database/cassandra/cassandraConnector.js

```diff
@@ -75,6 +81,30 @@ function getStats(testId, reportId) {
     return executeQuery(GET_REPORT_STATS, params, queryOptions);
 }
 
+function subscribeRunner(testId, reportId, runnerId) {
+    let params;
+    params = [testId, reportId, runnerId, constants.SUBSCRIBER_INITIALIZING_STAGE];
```

Comment:
dont forget to move the const to the manager

Reply:
done